src/rgw/rgw_user.cc

```diff
@@ -1850,6 +1861,12 @@ int RGWUser::check_op(RGWUserAdminOpState& op_state, std::string *err_msg)
     return -EINVAL;
   }
 
+  if (rgw_is_valid_tenant(op_id.tenant)) {
```

Comment:
well the only issue with that is that `rgw_admin` would exit with a not so relevant exit code (instead of -EINVAL)